src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,66 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    public static int splitAndSum(String text) {
+        if (isBlank(text))
+            return 0;
```

Comment:
```suggestion
        if (isBlank(text)) {
            return 0;
        }
```

[구글의 Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used) 를 따라주시면 감사하겠습니다.
구글의 Java Style Guilde 의 경우 brace 는 항상 붙이기를 권장하고 있습니다.

Reply:
네 붙이도록 할게요!